site/gatsby-site/server/fields/entity_relationships.ts

```diff
@@ -0,0 +1,28 @@
+import { GraphQLFieldConfigMap, GraphQLObjectType } from "graphql";
+import { allow } from "graphql-shield";
+import { generateMutationFields, generateQueryFields } from "../utils";
+import { Context } from "../interfaces";
+import { Entity_relationshipType } from "../types/entity_relationship";
+
+
+export const queryFields: GraphQLFieldConfigMap<any, Context> = {
+
+    ...generateQueryFields({ collectionName: 'entity_relationships', Type: Entity_relationshipType })
+}
+
+
+export const mutationFields: GraphQLFieldConfigMap<any, Context> = {
+
+    ...generateMutationFields({ collectionName: 'entity_relationships', Type: Entity_relationshipType, generateFields: ['updateOne', 'upsertOne', 'deleteOne'] }),
+}
+
+export const permissions = {
+    Query: {
+        entity_relationships: allow,
+    },
+    Mutation: {
+        updateOneEntity_relationship: allow,
```

Comment:
One thing to note is that we don't have to stick to these mutations anymore, especially if we are performing more than one operation in a UI flow, i.e., it is better to have one `promoteSubmission` mutation that touches multiple collections than relying on separate small mutations like these. (This might not be the case just flagging this up)